stdlib/public/core/Runtime.swift.gyb

```diff
@@ -408,37 +408,17 @@ internal func _float${bits}ToString(
   _ value: Float${bits}, debug: Bool
```

Comment:
I've kept the `debug` flag here even though it's not used.  The only difference between `debugDescription` and `description` is how they print NaNs.  That difference is now entirely encapsulated -- `description` checks for and returns the static string immediately; the bottom C layer always produces the debug form, so `debugDescription` can just pass NaNs down.